jsapp/js/account/accountFieldsEditor.component.tsx

```diff
@@ -13,7 +13,7 @@ import type {
 } from './account.constants';
 
 // See: kobo/apps/accounts/forms.py (KoboSignupMixin)
-const ORGANIZATION_TYPE_SELECT_OPTIONS = [
+export const ORGANIZATION_TYPE_SELECT_OPTIONS = [
```

Comment:
IMO it's a bit off to have this exported from "accountFieldsEditor", maybe we should move this const somewhere else more directed to organizations?